test/executor_test.exs

```diff
@@ -239,7 +239,7 @@ defmodule WaspVM.ExecutorTest do
       {:ok, pid} = WaspVM.start()
       WaspVM.load_file(pid, "test/fixtures/wasm/nested_func_call.wasm")
 
-      {:ok, gas, result} =  WaspVM.execute(pid, "nested_func_call", [0, 32])
+      {:ok, gas, result} =  WaspVM.execute(pid, "nested_func_call", [0, 32], [gas_limit: :infinity])
```

Comment:
Gas limit defaults to `:infinity` if it isn't passed IIRC, so this doesn't need to be here

Reply:
Fixed, Removed